appveyor/install-miniconda.ps1

```diff
@@ -76,8 +76,12 @@ $env:PATH = "${env:PYTHON};${env:PYTHON}\Scripts;" + $env:PATH
 
 # Conda config
 conda config --set always_yes true
-conda config --add channels astropy-ci-extras
 
+if ($conda_channels -ne $null) {

```

Comment:
Also it seems to me that this conditional should be the other way around, as it goes into the ``else`` part if no ``conda_channels`` was provided